### DISCUSSION TITLE
Dashboard for the NsPanel Pro

### DISCUSSION THREAD

**Discussion by Godferdom** [Blackymas/NSPanel_HA_Blueprint#959] [Ideas]
Hi,

This seems a really beautiful work!
I was wondering if it would be possible to create a dashboard for the NsPanel Pro based on this project? With the possibility to navigate between the screens by swiping.

Thanks for this project!
Regards,
JC

**Comment by edwardtfn** [comment]
Unfortunately the NSPanel is completely different from this one (it's isn't ESP32 based) and would require a completely different implementation.

**Comment by Godferdom** [comment]
Hi Edward,

Indeed the pro is working on android 8.1. It can be used with the HA Companion. So I was thinking if your Dashboard style couldn't be copied as an classical dashboard and imported?
